include/uv.h

```diff
@@ -512,6 +680,14 @@ UV_EXTERN int uv_fileno(const uv_handle_t* handle, uv_os_fd_t* fd);
 
 UV_EXTERN uv_buf_t uv_buf_init(char* base, unsigned int len);
 
+#if defined(_MSC_VER)
+# define INLINE __inline
+#elif defined(__GNUC__) || defined(__MVS__)
+# define INLINE __inline__
+#else
+# define INLINE inline
+#endif
```

Comment:
You can probably use `inline` directly, seeing we're C99/C11 now. It can't be called `INLINE` at any rate, too much risk of that already being defined; it would have to be something like `UV_INLINE`.

Reply:
This is pre-existing in v2, just moved from there. But I guess using `inline` keyword now is hopefully correctly supported everywhere?